src/replay.c

```diff
@@ -125,7 +125,7 @@ replay_file(tcpreplay_t *ctx, int idx)
 
     /* read from pcap file if we haven't cached things yet */
     if (!ctx->options->preload_pcap) {
-        if ((pcap = pcap_open_offline(path, ebuf)) == NULL) {
+        if ((pcap = pcap_open_offline_with_tstamp_precision(path, PCAP_TSTAMP_PRECISION_NANO, ebuf)) == NULL) {
```

Comment:
This is a bit misleading function, in case I use this
nanoseconds will be stored in tv_usec value of a timeval struct. 

https://stackoverflow.com/questions/17636334/read-nanosecond-pcap-file-using-libpcap